app/config/FrontendAppConfig.scala

```diff
@@ -118,8 +118,10 @@ class FrontendAppConfig @Inject()(val servicesConfig: ServicesConfig, val config
   lazy val calcPollSchedulerTimeout: Int = servicesConfig.getInt("calculation-polling.timeout")
 
   // Submission service
-  lazy val submissionUrl: String = servicesConfig.baseUrl("income-tax-submission-frontend")
-  lazy val fullSubmissionUrl: String = s"$submissionUrl/income-through-software/return/2022/start"
+  lazy val submissionFrontendUrl: String = {
+    s"${servicesConfig.getString("income-tax-submission-frontend.url")}/income-through-software/return/2022/start"
```

Comment:
we do not need the string appending at the end i.e., everything needs to be in config.